docs/developer-manual/consume.md

```diff
@@ -4,31 +4,248 @@ sidebar_position: 1
 slug: /consume
 ---
 
-# Consume fact statements
+# Consuming Orcfax Statements
 
-## Important Notice
+This document aims to explain how dapp developers (ie integrators) can integrate
+Orcfax feeds into their dapp with all the gory (technical) details included.
 
-This page is being updated to reflect the upcoming changes in the Orcfax v1
-solution. Once this upgrade has been released, instructions on how to consume
-the enhanced datum will be made available here.
+## Overview
 
-Historical data concerning the v0 Orcfax solution can be found in
-[Archived resources](archived-resources) and information pertaining to policy
-ID's in [Policies](policies).
+### Orcfax
 
-## Fact statements
+Orcfax is a Cardano native oracle platform.
 
-A fact statement is a datum that the Orcfax network publishes to a Cardano
-blockchain transaction. This datum can then be used as an input to Cardano smart
-contracts and dApps.
+Orcfax is organized into feeds. Generally a feed concerns a time series data,
```

Comment:
"...concerns a time series [of] data" ?

Reply:
I don't think of is necessary. See first line here https://en.wikipedia.org/wiki/Time_series#Models

Reply:
But note that in your example it doesn't read
> "Models for [a] time series data

I can not quote the rule, but I'm pretty sure the "a" needs to disappear 
> Generally a feed concerns time series data

Or "of" needs to follow
> Generally a feed concerns a time series of data

Reply:
I'll take the first option